plugins/woocommerce/client/blocks/package.json

```diff
@@ -264,6 +264,7 @@
 		"@emotion/styled": "^11.11.0",
 		"@preact/signals": "^1.3.0",
 		"@woocommerce/tracks": "workspace:*",
+		"@wordpress/a11y": "4.22.0",
```

Comment:
Needed this as a dependency otherwise pnpm would doggedly try to resolve the dependency from relative path regardless of what I did in `readPackage`